src/Hosting/TestHost/test/ClientHandlerTests.cs

```diff
@@ -421,7 +420,7 @@ public Task ExceptionFromOnStartingWithErrorHandlerIsReported()
                 HttpCompletionOption.ResponseHeadersRead));
         }
 
-        private class DummyApplication : IHttpApplication<Context>
+        private class DummyApplication : ApplicationWrapper, IHttpApplication<HostingApplication.Context>
```

Comment:
This is the last thing to clean up or is this a test so it's fine?

Reply:
Just a test, I'm not worried about it.

Reply:
Sure, it's pretty horrible though

Reply:
Fixed